test/OpenTelemetry.Tests/Impl/Metrics/MetricsTest.cs

```diff
@@ -55,39 +55,6 @@ public void CounterSendsAggregateToRegisteredProcessor()
             Assert.Equal(210, testProcessor.counters[0].Item3);
         }
 
-        [Fact(Skip = "TODO: Make robust tests. These are just basic tests to see metric in action.")]
```

Comment:
please add a test for observer?

Reply:
Added test for Observer, and fixed/enabled the tests for measure & counter.